packages/storybook-react/src/stories/prototype-src/ContentWebpage.tsx

```diff
@@ -0,0 +1,213 @@
+import {
+  AccordionProvider,
+  ButtonGroup,
+  ButtonLink,
+  Checkbox,
+  Heading1,
+  Heading2,
+  Heading3,
+  Heading4,
+  Link,
+  LinkList,
+  LinkListLink,
+  PageFooter,
+  Paragraph,
+  RadioButton,
+  SpotlightSection,
+  Strong,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react/dist/css-module';
+import {
+  UtrechtIconArrow,
+  UtrechtIconChevronLeft,
+  UtrechtIconChevronRight,
+  UtrechtIconFacebook,
+  UtrechtIconInstagram,
+  UtrechtIconLinkedin,
+  UtrechtIconNieuwsbrief,
+  UtrechtIconWhatsapp,
+} from '@utrecht/web-component-library-react';
+import React from 'react';
+import urls from './variables.js';
+const ContentWebpage: React.FC = () => {
+  return (
+    <>
+      <Heading1>Testtekst voor Font Vergelijking</Heading1>
+      <Heading2>Leesbaarheid & Lettercombinaties</Heading2>
+      <Paragraph appearance="lead">
+        Het is belangrijk om te zien hoe een <strong>lettertype omgaat met lettercombinaties</strong> die visueel op
```

Comment:
Zou je graag de `Strong`-component uit het NL Design System willen gebruiken in plaats van de HTML `<strong>`-tag, niet alleen hier maar overal?